src/components/Header.js

```diff
@@ -1,10 +1,57 @@
-import React from 'react';
-import styled from 'styled-components';
+import React from "react";
+import styled from "styled-components";
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
+export default function Header({ changeUser, newMessage }) {
+  const EUNKO = "https://img.techpowerup.org/200908/eun.png";
+  const COOL = "https://img.techpowerup.org/200908/NjRiY2JjOGU5YzQz.png";
+  const userCheck = () => {
+    if (newMessage.user) {
+      return <Image src={EUNKO} />;
+    } else {
+      return <Image src={COOL} />;
+    }
+  };
+
+  const displayName = () => {
+    if (newMessage.user) {
+      return "고은";
+    } else {
+      return "정쿨";
+    }
+  };
+
+  return (
+    <Wrapper>
+      <ChangeButton onClick={changeUser}>{userCheck()}</ChangeButton>
+      <ShowUser>
+        <UserName>{displayName()}</UserName>
+        <UserOn>현재 활동중</UserOn>
+      </ShowUser>
+    </Wrapper>
+  );
 }
 const Wrapper = styled.div`
-	background-color: black;
-	color: white;
+  width: 100vw;
+  background-color: rgba(255, 255, 255, 0.8);
+  display: flex;
+  position: fixed;
+  width: 100%;
+`;
+const ShowUser = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+`;
+const UserName = styled.div`
+  font-size: 1.8rem;
+`;
+
+const UserOn = styled.div``;
+
+const ChangeButton = styled.div``;
+
+const Image = styled.img`
+  width: 5rem;
```

Comment:
height의 default값은 auto인데, 원본이미지파일의 height자체가 달라서,
 유저가 바뀔때마다 헤더의 크기가 달라져요 ㅠㅠ
height도 5rem으로 설정해서 고정시켜주시면 좋을 것 같아요!
```suggestion
  width: 5rem;
  height:5rem;
```

Reply:
아하 이 부분은 제가 제대로 이해를 못했던 것 같아요 알려주셔서 감사합니다!

Reply:
저도 신경쓰였던 부분인데 이런 해결책이 있군요